language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -201,4 +202,30 @@ impl VerifierConfig {
             ..VerifierConfig::default()
         }
     }
+
+    /// An approximation of what config is used in production.
+    pub fn production() -> Self {
+        Self {
+            max_loop_depth: Some(5),
+            max_generic_instantiation_length: Some(32),
+            max_function_parameters: Some(128),
+            max_basic_blocks: Some(1024),
+            max_basic_blocks_in_script: Some(1024),
+            max_value_stack_size: 1024,
+            max_type_nodes: Some(256),
+            max_push_size: Some(10000),
+            max_dependency_depth: Some(100),
+            max_struct_definitions: Some(200),
+            max_fields_in_struct: Some(30),
+            max_function_definitions: Some(1000),
+
```

Comment:
This won't conflict with anything that already exists right?

Reply:
It replaces some config used for testing locally to become available globally.